### PR TITLE
feat: Add types to the Migrator interface

### PR DESCRIPTION
**Pull request by lffg**
Add typings to the `list` and `forceFreeMigrationsLock` methods.

### CODE REVIEW

File: types/index.d.ts [hunk 1]
```diff
@@ -1828,6 +1828,8 @@ declare namespace Knex {
     currentVersion(config?: MigratorConfig): Promise<string>;
     up(config?: MigratorConfig): Promise<any>;
     down(config?: MigratorConfig): Promise<any>;
+    list(config?: MigratorConfig): Promise<any>;
```
Comment (kibertoad) [inline types/index.d.ts]: I think list is already included and can be removed from this PR?

Comment (lffg) [reply types/index.d.ts]: Where?

Comment (kibertoad) [reply types/index.d.ts]: https://github.com/knex/knex/blob/master/types/index.d.ts#L1830

Comment (lffg) [reply types/index.d.ts]: Ah, ok, thanks. :)

**Comment by kibertoad** [comment]
Thanks!

**Pull request merged**